terraform/deployments/cluster-services/aws_auth_configmap.tf

```
@@ -61,3 +76,47 @@ resource "kubernetes_cluster_role_binding" "cluster_admins" {
     api_group = "rbac.authorization.k8s.io"
   }
 }
+
+resource "kubernetes_cluster_role_binding" "cluster_readonly" {
+  metadata {
+    name = "cluster-readonly"
+  }
+  role_ref {
+    api_group = "rbac.authorization.k8s.io"
+    kind      = "ClusterRole"
+    name      = "view"
+  }
+  subject {
+    kind      = "Group"
+    name      = "readonly"
+    api_group = "rbac.authorization.k8s.io"
+  }
+}
+
+resource "kubernetes_cluster_role" "additional_cluster_readonly" {
```

Can we call this something more informative? (and update the k8s object name to match)
```suggestion
resource "kubernetes_cluster_role" "read_crs_and_crbs" {
```

---

yes, I can do that.